CalendarApp/View Controllers/DailyCalendarViewController.swift

```diff
@@ -23,47 +23,49 @@ class DailyCalendarViewController : DayViewController {
     private let calendarEventHandler = CalendarEventHandler()
     private var fetchedDates = Set<Date>()
     
-    func updateCalendarEvent(_ event: CalendarApp.Event, _ originalStart: Date, _ newStart: Date) {
-        calendarEventHandler.updateEvent(event, originalStart, newStart)
+    func updateCalendarEvent(_ event: CalendarApp.Event, originalStartDate originalStart: Date, originalEndDate originalEnd: Date) {
+        calendarEventHandler.updateEvent(event, originalStart, originalEnd)
         reloadData()
     }
     
-    func deleteCalendarEvent(_ event: CalendarApp.Event, _ date: Date) {
-        calendarEventHandler.deleteEvent(event, date)
+    func deleteCalendarEvent(_ event: CalendarApp.Event) {
+        calendarEventHandler.deleteEvent(event)
         reloadData()
     }
     
     func failedRequest(_ errorMessage: String) {
         // TODO: Error handling
     }
     
-    func addEvent(_ eventModel: CalendarApp.Event, _ date: Date) {
-        calendarEventHandler.addEvent(eventModel, date)
+    func addEvent(_ eventModel: CalendarApp.Event) {
```

Comment:
This API is not named consistently like the others (update, delete, etc.). e.g: `deleteCalendarEvent` has `Calendar` in the method name and also has `event` as a parameter name, as opposed to this one that has no `Calendar` and the parameter name is `eventModel`